command/crypto/key/sign.go

```diff
@@ -237,3 +256,26 @@ func rsaHash(ctx *cli.Context) (crypto.SignerOpts, error) {
 
 	return h, nil
 }
+
+func readKey(keyFile string, isPubKey bool, ctx *cli.Context) (interface{}, error) {
+	if strings.ToLower(ctx.String("alg")) == "es256k" {
+		hexRaw, err := os.ReadFile(keyFile)
+		if err != nil {
+			return nil, errors.Wrap(err, "read file error")
+		}
+		raw, err := hex.DecodeString(strings.TrimPrefix(strings.TrimSpace(string(hexRaw)), "0x"))
+		if err != nil {
+			return nil, errors.Wrap(err, "file content is not in hex")
+		}
+		if isPubKey {
+			secp256k1Pk, err := secp256k1.ParsePubKey(raw)
+			if err != nil {
+				return nil, errors.Wrap(err, "unable to parse public key")
+			}
+			return secp256k1Pk.ToECDSA(), nil
+		}
+		secp256k1Pk := secp256k1.PrivKeyFromBytes(raw)
+		return secp256k1Pk.ToECDSA(), nil
+	}
```

Comment:
Is there any reason to not use a PEM file for this?
```console
$ openssl ecparam -name secp256k1 -genkey -noout
-----BEGIN EC PRIVATE KEY-----
MHQCAQEEIMUALqrFTe1KusIe3WlCZeRHBZoNoL1SbqzVDHdHo+7roAcGBSuBBAAK
oUQDQgAEkQ/Pj3MnBvwmNmsYEg0cCqgsXwV8yKYJHG099jfLPjTdmV3ZWkZg146Q
Nfm0RBXjvgEoVXhHy/g2vyptMmAaKQ==
-----END EC PRIVATE KEY-----
```

Reply:
No, there's no reason. We can certainly use PEM here.

Maybe we can add support for hex-encoded key and wallet (JSON, imported format). The support can be done directly in related commands or in the `format` command.

Blockchain client node implementations often import keys via those above methods.